arxiv/arxiv.py

```diff
@@ -233,12 +233,22 @@ def slugify(obj):
     return filename
 
 
-def download(obj, dirpath='./', slugify=slugify):
+def download(obj, dirpath='./', slugify=slugify, source=False):
```

Comment:
I'd prefer a more descriptive argument name here; `source` has multiple meanings in the context of a download, and (as discussed below) this might still yield a PDF download.

Suggestion: `prefer_source_tarfile=False`.